test/utils/image_utils.py

```diff
@@ -39,12 +39,10 @@ def build_image(framework_name, framework_version, py_version, processor, tag, c
 
     image_uri = get_image_uri(framework_name, tag)
 
-    dockerfile_location = os.path.join('docker', framework_version, 'final',
-                                       'Dockerfile.{}'.format(processor))
+    dockerfile_location = os.path.join('test-toolkit', 'docker', 'Dockerfile.gpu')
```

Comment:
I thought you were leaving the `test/` directory untouched for this change?

Reply:
we were, oops.